core/src/main/java/com/gentics/mesh/changelog/highlevel/HighLevelChangesList.java

```diff
@@ -1,40 +1,20 @@
 package com.gentics.mesh.changelog.highlevel;
 
-import java.util.Arrays;
 import java.util.List;
 
-import javax.inject.Inject;
 import javax.inject.Singleton;
 
-import com.gentics.mesh.changelog.highlevel.change.ExtractPlainText;
-import com.gentics.mesh.changelog.highlevel.change.SetAdminUserFlag;
 import com.gentics.mesh.core.data.changelog.HighLevelChange;
 
 /**
  * List of high level changes which can be applied once the low level changelog entries have been applied.
  */
 @Singleton
-public class HighLevelChangesList {
+public interface HighLevelChangesList {
```

Comment:
Maybe it would be more flexible to make this class abstract and leave the (common) high level changes in here?
Context specific implementations (like OrientDBHighLevelChangesList) should just make sure, that they call super.getList() in their getList() implementation and add the DB specific changes to the common list.
Currently, 50% of the high-level changes we have would apply for both the OrientDB and the MDM version.

Reply:
If we will have common high-level changes, than yes, it makes sense. I will implement that change

Reply:
done